client/src/featureform/local_cache.py

```diff
@@ -0,0 +1,169 @@
+import json
+import os
+import re
+from functools import lru_cache
+from typing import Callable, Set
+
+import pandas as pd
+from featureform import SQLiteMetadata  # fix to do client.source.import
+from featureform.resources import SourceType  # fix to do client.source.import
+from pandas.core.generic import NDFrame
+from typeguard import typechecked
+
+
+class LocalCache:
+    def __init__(self, db: SQLiteMetadata):
+        self.db = db
+        self.cache_dir = os.environ.get("FEATUREFORM_CACHE_DIR", ".featureform/cache")
```

Comment:
this should possibly be elsewhere. We need an env var for .featureform directory so that this is a relative path

Reply:
Can you make the featureform directory an env var then set the default to ~/.featureform and set this inside of that?